tools/expotools/src/Packages.ts

```diff
@@ -0,0 +1,102 @@
+import path from 'path';
+import glob from 'glob';
+import fs from 'fs-extra';
+import * as Directories from './Directories';
+
+const PACKAGES_DIR = Directories.getPackagesDir();
+
+/**
+ * Represents a package in the monorepo.
+ */
+class Package {
+  path: string;
+  packageJson: any;
+  unimoduleJson: any;
+
+  constructor(path: string, packageJson: { [key: string]: any }) {
+    this.path = path;
+    this.packageJson = packageJson;
+    this.unimoduleJson = readUnimoduleJsonAtDirectory(path);
+  }
+
+  get packageName(): string {
+    return this.packageJson.name;
+  }
+
+  get packageVersion(): string {
+    return this.packageJson.version;
+  }
+
+  get packageSlug(): string {
+    return this.unimoduleJson && this.unimoduleJson.name || this.packageName;
+  }
+
+  get scripts(): { [key: string]: string } {
+    return this.packageJson.scripts || {};
+  }
+
+  get podspecName(): string | null {
+    if (!this.unimoduleJson) {
+      return null;
+    }
+
+    const iosConfig = { subdirectory: 'ios', ...('ios' in this.unimoduleJson ? this.unimoduleJson.ios : {}) };
```

Comment:
FYI this can just be `{ subdirectory: 'ios', ...this.unimoduleJson.ios };` (`...null` / `...undefined` is fine).